gef-ui/server/routes/amend-facility.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const { validateRole, validateToken } = require('../middleware');
+const CONSTANTS = require('../constants');
+const { whatDoYouNeedToChange } = require('../controllers/amend-facility/what-do-you-need-to-change');
+
+const router = express.Router();
+
+router.get(
+  '/amend-facility/what-do-you-need-to-change',
+  [validateToken, validateRole({ role: [CONSTANTS.USER_ROLES.MAKER] })],
```

Comment:
Probably a question for @abhi-markan - should the `Admin` role also be able to see everything that a maker can?

Reply:
admin can see everything, but can't make changes to a deal/facility. so readOnly access level for Admins